skin.json

```diff
@@ -240,7 +240,8 @@
       {"name":"stereoscopic", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":45 },
       {"name":"fullscreen", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":45 },
       {"name":"moreOptions", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":45 },
-      {"name":"arrowsBlack", "location": "mainView", "whenDoesNotFit":"keep", "minWidth":45 }
+      {"name":"arrowsBlack", "location": "mainView", "whenDoesNotFit":"keep", "minWidth":45 },
+      {"name":"multiAudio", "location": "controlBar", "whenDoesNotFit":"keep", "minWidth":45 }
```

Comment:
Does this new button need to be added to the `buttonArray` in the `skin-schema.json`?

Reply:
Do we also need to update ``buttonNames`` in ``skin-schema.json``?

Reply:
@jpfranco 
Sorry, I did not understand what we need to add to buttonArray, because buttonArray is
```javascript
    "buttonArray": {
      "description": "Used to list the buttons for a single platform (desktop vs. mobile).",
      "type": "array",
      "items": { "$ref": "#/definitions/button" }
    },
```

Reply:
@aeng7 I added "multiAudio" to buttonNames in skin-schema.json.

Reply:
@KseniiaSokolovaEpam sorry, I was referring to the `buttonNames`